<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lostpassword</class>
 <widget class="QMainWindow" name="lostpassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>246</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #18191c;
font: 600 12pt &quot;Segoe UI&quot;;
color: #fff;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font-weight: 700;
border: none;</string>
        </property>
        <property name="text">
         <string>Логин</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="login_user">
        <property name="styleSheet">
         <string notr="true">background-color:  #383838;
border: 2px solid #b3d4fc;
border-radius: 5px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font-weight: 700;
border: none;</string>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="email_user">
        <property name="styleSheet">
         <string notr="true">background-color:  #383838;
border: 2px solid #b3d4fc;
border-radius: 5px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="Start_Button">
        <property name="styleSheet">
         <string notr="true">background-color:  #383838;
border: 2px solid #b3d4fc;
border-radius: 5px;
color: #fff;</string>
        </property>
        <property name="text">
         <string>Отправить</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QProgressBar" name="progressBar">
        <property name="styleSheet">
         <string notr="true">background-color:  #383838;
border: 2px solid #b3d4fc;
border-radius: 5px;</string>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labe_prog_value">
        <property name="styleSheet">
         <string notr="true">background-color:  #383838;
border: 2px solid #b3d4fc;
border-radius: 5px;
width: 10px;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QTextEdit" name="info_pass">
      <property name="styleSheet">
       <string notr="true">background-color:  #383838;
border: 2px solid #b3d4fc;
border-radius: 5px;
color: rgb(255, 0, 0);
font: 600 12pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="lostpass_exit">
      <property name="styleSheet">
       <string notr="true">background-color:  #383838;
border: 2px solid #b3d4fc;
border-radius: 5px;
color: #fff;</string>
      </property>
      <property name="text">
       <string>Выйти</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>lostpass_exit</zorder>
   <zorder></zorder>
   <zorder></zorder>
   <zorder>info_pass</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);
font: 600 12pt &quot;Segoe UI&quot;;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
